adserver/templates/adserver/advertiser/advertisement-detail.html

```diff
@@ -49,6 +49,8 @@ <h1>{% block heading %}{% trans 'Advertisement: ' %}{{ advertisement.name }}{% e
       <dd>{{ advertisement.flight.name }}</dd>
       <dt>{% trans 'Display' %}</dt>
       <dd>{{ advertisement.ad_types.all | join:"<br>" }}</dd>
+      <dt><abbr title="{% trans 'Click through rate' %}">{% trans 'CTR' %}</abbr></dt>
+      <dd>{{ advertisement.ctr|floatformat:3 }}%</dd>{# Ad CTR actually does a full calculation #}
```

Comment:
I don't fully understand this comment. Are you saying it's computing something expensive?

Reply:
It's computing the CTR. It isn't that expensive when doing this for a single ad. If you did this for a bunch of ads (this doesn't do that), then yes, it could get expensive.